### DISCUSSION TITLE
Users can not download purchases product form customer dashboard

### DISCUSSION THREAD

**Discussion by ssdarjun** [bagisto/bagisto#7156] [Announcements]
## Issue Description
Order status completed and invoice status also paid but if I want to download a downloadable product from customer dashboard then show a warning message- "Warning! Payment has not been done for this download"

## Bagisto Version
- [ ] 1.4.4

**Comment by Sanjaybhattwebkul** [comment]
Hi there,
Is it **possible** to share the **screen recording** for the same because I am **not getting** this issue in our **V1.4.4?**

**Comment by ssdarjun** [comment]
Can we join in zoom/google meet?

**Comment by ssdarjun** [comment]
Please check this video using bellow link. I hope you will understand. If still not understand please let me know. thank you

https://drive.google.com/file/d/1M8QXGBUkpbdiyrOYJy10GgP_VRHbmSsG/view?usp=sharing

**Comment by Sanjaybhattwebkul** [comment]
Hi there,
**Thanks** for reporting, we will **fix** this issue in our **next release.**

**Comment by Sanjaybhattwebkul** [comment]
> Please check this video using bellow link. I hope you will understand. If still not understand please let me know. thank you
> 
> https://drive.google.com/file/d/1M8QXGBUkpbdiyrOYJy10GgP_VRHbmSsG/view?usp=sharing

Actually, the transaction is not created for the order so the customer is getting the warning message.
